awswrangler/catalog/_create.py

```diff
@@ -970,7 +970,7 @@ def create_json_table(
     description: Optional[str] = None,
     parameters: Optional[Dict[str, str]] = None,
     columns_comments: Optional[Dict[str, str]] = None,
-    mode: str = "overwrite",
+    mode: Literal["overwrite", "append"] = "overwrite",
```

Comment:
Nitpick - shall we have a type var for this as well to avoid copy-paste like in the other files?

Reply:
I thought about this but honestly this list values for `mode` changes quite a bit from function to function. The `update_ruleset` function has `overwrite` and `upsert`. A lot of the S3 methods have `append`, `overwrite` and `overwrite_partitions`.